tests/phpunit/TemplateTest.php

```diff
@@ -204,4 +208,30 @@ public function testErrantAuthor() {
     $this->assertEquals($text, $expanded->parsed_text());
   }
 
+  public function testInPress() {
+    $text = '{{Cite journal|pmid=9858585|date =in press}}';
+    $expanded = $this->process_citation($text);
+    $this->assertNull($expanded->get('date'));
+    $this->assertEquals('1999',$expanded->get('year'));
+
+    $text = '{{cite journal|pmid=9858585|year=in press}}';
+    $expanded = $this->process_citation($text);
+    $this->assertNull($expanded->get('date'));
+    $this->assertEquals('1999',$expanded->get('year'));
+  }
+  
+  public function testSICI() {
```

Comment:
Does `testSICI` test anything that is not covered in `testSiciExtraction()` (line 146)?

Reply:
Missed that one